client/controllers/deactivateUsersCtrl.js

```diff
@@ -19,5 +33,19 @@ module.exports = [
 
       $route.reload();
     };
+
+    $scope.deleteRegister = function (register) {
+      if (confirm('Er du sikker på at du vil slette denne brukeren?')) {
+        registerService.deleteRegisterEntry(register).then(
+          function (response) {
+            alertService.addSuccess(response.data.message);
+            $route.reload();
```

Comment:
Is there really no better way to do this?

Reply:
One can pop the element or call the function again, but the full reload seemed safer.